project/Dependencies.scala

```diff
@@ -40,7 +40,7 @@ object Dependencies {
   object Compile {
     // Compile
 
-    val camelCore     = "org.apache.camel"            % "camel-core"                   % "2.13.4" exclude("org.slf4j", "slf4j-api") // ApacheV2
+    val camelCore     = "org.apache.camel"            % "camel-core"                   % "2.17.1" exclude("org.slf4j", "slf4j-api") // ApacheV2
```

Comment:
Thanks, but i'm still not clear with respect to dependencies. Are we after:
1. binary compatibility of all types in 3rd party dependencies?
2. binary compatibility with camel-core classes/types used by akka-camel
3. binary compatibility with camel-core classes/types used directly by akka-camel and exposed through akka-camel
or something else?